lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -38,9 +38,19 @@ def descriptions(version)
     end
 
     def all_questions
-      dataset.keys.inject({}){|versions, version| versions.merge(questions(version))}
+      dataset.keys.inject({}) { |versions, version| versions.merge(questions(version)) }
     end
 
+    def is_london?(postcode)
+      area(postcode).any? { |result| result[:slug] == "london" }
```

Comment:
I agree, though I think the "is" more readable. Changed in commit: fbe1f8029110df2f91a5cbf039b7a94ed1ebb8cc